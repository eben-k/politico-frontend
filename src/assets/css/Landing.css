h3, h4 {text-align: center}
.container {width: 100%; margin: 0px auto;}
[class*="col-"] {float: left; padding: 15px;}
.col-1 {width: 8.33%;}
.col-2 {width: 16.66%;}
.col-3 {width: 26%;}
.col-4 {width: 33.33%;}
.col-5 {width: 41.66%;}
.col-6 {width: 50%;}
.col-7 { width: 58.33%;}
.col-8 {width: 66.66%;}
.col-9 {width: 75%;}
.col-10 {width: 83.33%;}
.col-11 {width: 91.66%;}
.col-12 {width: 100%;}

.catch-phrase {
  text-align: center
}

.card {
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  transition: 0.3s;
  color: rgba(0, 181, 204, 1);
  margin: 15px;
  position: relative;

}
.card h4{
  float: left;
  /* display: inline-block; */
}
.text{
  float: right;
  /* display: inline-block; */

}
.card .col-3{
  width: 30%;
}
.text-left{
  float:left;
  /* display: inline-block; */

}
#page{
 padding-left: 90px;
 justify-content: center;
 height: 100vh;
 width: 92%;;
 margin-left: auto;
 margin-right: auto;
}
#page::before, #page::after{
  clear: both;
}
.card img {
  display: block;
  width: 100%;
  margin-left: auto;
  margin-right: auto;
  height: 180px;

}

.card:hover {
  box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
  border-radius: 10px;
}

.card-detail{
  padding: 2px 16px;

}

@media screen and (max-width: 850px){
  [class*="col-"] {

      padding: 15px;
      width: 65%;
      margin-left: auto;
      margin-right: auto;
  }
}
