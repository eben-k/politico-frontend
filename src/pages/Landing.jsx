import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import Layout from '../components/presentationals/Layout';

import '../assets/css/Landing.css';
import general from '../assets/images/general.jpg';
import vote from '../assets/images/vote.png';
import vote3 from '../assets/images/vote-3.jpg';
import ballot from '../assets/images/ballot.png';
import politics2 from '../assets/images/politics-24.jpg';
import administration from '../assets/images/administration.jpg';
class Homepage extends Component {
  render() {
    return (
      <Layout>
        <div>
          <h1 className="catch-phrase"> Your Vote Always Counts!!! </h1>
        </div>
        <div id="page">
          <div className="container">
            <div className="card col-3">
              <img src={general} alt="Avatar" />
              <div className="card-detail-">
                <Link
                  to="/signup"
                  style={{
                    textDecoration: 'none',
                  }}
                >
                  <h3>Register To Vote</h3>
                </Link>
              </div>
            </div>
          </div>

          <div className="container">
            <div className="card col-3">
              <img src={vote} alt="Avatar" />
              <div className="card-detail-">
                <Link
                  to="/parties"
                  style={{
                    textDecoration: 'none',
                  }}
                >
                  <h3>Political Parties</h3>
                </Link>
              </div>
            </div>
          </div>
          <div className="container">
            <div className="card col-3">
              <img src={vote3} alt="Avatar" />
              <div className="card-detail-">
                <Link
                  to="/"
                  style={{
                    textDecoration: 'none',
                  }}
                >
                  <h3>Candidates</h3>
                </Link>
              </div>
            </div>
          </div>
          <div className="container">
            <div className="card col-3">
              <img src={ballot} alt="Avatar" />
              <div className="card-detail-">
                <Link
                  to="/"
                  style={{
                    textDecoration: 'none',
                  }}
                >
                  <h3>Voter Resources</h3>
                </Link>
              </div>
            </div>
          </div>
          <div className="container">
            <div className="card col-3">
              <img src={politics2} alt="Avatar" />
              <div className="card-detail-">
                <Link
                  to="/"
                  style={{
                    textDecoration: 'none',
                  }}
                >
                  <h3>Election Data</h3>
                </Link>
              </div>
            </div>
          </div>
          <div className="container">
            <div className="card col-3">
              <img src={administration} alt="Avatar" />
              <div className="card-detail-">
                <Link
                  to="/"
                  style={{
                    textDecoration: 'none',
                  }}
                >
                  <h3>Poll Workers</h3>
                </Link>
              </div>
            </div>
          </div>
        </div>
      </Layout>
    );
  }
}
export default Homepage;
