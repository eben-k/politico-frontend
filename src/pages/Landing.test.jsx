import React from 'react';
import { shallow } from 'enzyme';

import Landing from './Landing';

describe('Navbar', () => {
  it('should render correctly', () => {
    const component = shallow(<Landing className="" />);
    expect(component).toMatchSnapshot();
  });
});
