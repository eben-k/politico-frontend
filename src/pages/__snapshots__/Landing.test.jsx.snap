// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar should render correctly 1`] = `
<div>
  <h1
    className=""
  >
     Politico React App 
  </h1>
</div>
`;
